index.html

```diff
@@ -6,6 +6,22 @@
     <meta name="viewport" content="width=device-width,initial-scale=1.0" />
     <link rel="icon" href="/favicon.ico" />
     <title>Cuttle</title>
+
+    
+    <meta property="og:url" content="https://www.cuttle.cards/" />
+    <meta property="og:title" content="Cuttle Cards Online" />
+    <meta property="og:type" content="website" />
+    <meta property="og:image" content="/public/img/cuttle_meta_img.jpg/" />
+    <meta property="og:description"
+        content="Cuttle is a 2 player battle card game played with a standard 52-card deck of cards. It has the strategic nuance of trading card games like Magic, with the elegant balance of a standard deck--and you can play it for free!" />
+
+    <meta name="twitter:site" content="https://unacog.com/">
```

Comment:
Not sure what these do but they should point to the cuttle twitter, right?
```suggestion
    <meta name="twitter:site" content="https://twitter.com/cuttle_cards/">
```

Reply:
Yes, likely just copied from  somewhere else. I would prefer to have an env for this too.

Reply:
let me fix these, my mistake. 

We were suppose to replace https://unacog.com/ with https://cuttle.cards - this was from the code i pulled it from. 

Content should not be content="https://twitter.com/cuttle_cards/"> 
but content="https://cuttle.cards"> 

Twitter:site

Reply:
This does not point to cuttle twitter. it instead lets the OG protocol know what the site is about. Facebook made the protocol to make sharing more media rich. Twitter implemented their own standard and that's what this is pointing to. Discord uses the twitter meta data and so we need to put it in there if we want to have nice image and rich description when sharing the link on discord.

Reply:
Okay thanks for explaining. I haven't previously done much with metadata so now I see that `twitter` and `og` are essentially separate prefixes/protocols for defining the same properties in ways that will work across different media. Thanks for the tip.So this should be
```suggestion
    <meta name="twitter:site" content="https://cuttle.cards/">
```